packages/parser/common_auto_rollback.go

```diff
@@ -37,7 +37,8 @@ func (p *Parser) restoreUpdatedDBRowToPreviousData(tx map[string]string, where s
 	}
 	addSQLUpdate := ""
 	for k, v := range rollbackInfo {
-		if converter.InSliceString(k, []string{"hash", "pub", "tx_hash", "public_key_0", "node_public_key"}) && len(v) != 0 {
+		if ((k == "txhash" && converter.ValidateTable(tx["table_name"], `history`)) ||
```

Comment:
maybe it's better to use some map[string]string{"txhash": "history", "pub": "keys"}, than that copypasted hardcode

Reply:
Fixed. ValidateTable was refactored to IsByteColumn